### DISCUSSION TITLE
I want to add tag on repo too, not only add in list

### DISCUSSION THREAD

**Discussion by PubCyBerry** [community/community#9178] [Other Feature Feedback, Questions, & Ideas]
Sorting starred repos with lists is good idea.
But bad point of List system is it is hard to classify repos that have multiple tags.

for example, let's say a repo consists of 'python course about building web app'.
if someone has lists 'python', 'course', 'web app', where should it be?
I think all of them possible.

it may possible to handle it if you name lists very well.
but it is very hard and there are too many cases as starred repos get  increasing.

Plus I don't want Lists having repos overlapped each other.
If I could add tags on each folder, things become much easier.
with that above example, I add those 3 tags.

If I want repo that is related to 'python', 'course' and 'web app', I search repos which has those three tags.
But if there exists List system only, I have to find it in 'course' list or 'python' list or 'web app' list.

**Comment by brianlovin** [comment]
Thanks for the note @PubCyBerry — I agree it's not totally clear how topics and lists should interact. It could become more complex if we introduced more ways to tag and organize a list, but it sounds like you're trying to create very granular lists anyways so maybe the complexity would be okay. I'll pass this along to our team to think about 😊

**Reply by pbassham** [reply]
I came to suggest this same thing, but just realized that a repo can be in multiple lists, so that makes it work a little more like tags, which is helpful.

Some 'smart' features to the lists would be nice though, which seems like it would work better with a more tag-based approach.

So, a list could be a manual list OR a 'smart list' based on tags that you added or other features. 

Like, 
- smart list of all js/ts projects (based on language of repo)
- smart list based on #-of-stars rule (ex. repos with 1000+ stars)
- combo of tags (like OP's example)
- untagged (to help with comprehensive organization)

**Reply by brianlovin** [reply]
Love the smart list idea! Will pass that along to our team 😊

**Reply by QuincySx** [reply]
Although a repo can now be added to several lists, But multi-lists filtering is not supported in search.

**Reply by Yowyaz** [reply]
Cyber security
